ocs_samples/basic_samples/Authentication/ClientCredentialFlow/DotNet/ClientCredentialFlow/Program.cs

```diff
@@ -11,33 +11,49 @@ public static class Program
         public static void Main(string[] args)
         {
             InitConfig();
-                
+
             ClientFlow.OcsUrl = GetConfigValue("Resource");
 
             var tenantId = GetConfigValue("TenantId");
             var clientId = GetConfigValue("ClientId");
             var clientSecret = GetConfigValue("ClientKey");
+            var version = GetConfigValue("ApiVersion");
             ClientFlow.CreateAuthenticatedHttpClient(clientId, clientSecret);
 
             // Make an HTTP request to OCS using the authenticated client - since this is the first request, the AuthenticationHandler will
             // authenticate and acquire an Access Token and cache it.
-            var response = ClientFlow.AuthenticatedHttpClient.GetAsync($"api/Tenants/{tenantId}").Result;
-            response.EnsureSuccessStatusCode();
-            Console.WriteLine(response.Content.ReadAsStringAsync());
-            Console.WriteLine($"HTTP GET api/Tenants/{tenantId} successful");
+            try
+            {
+                var response = ClientFlow.AuthenticatedHttpClient.GetAsync($"api/Tenants/{tenantId}/Users").Result;
+                response.EnsureSuccessStatusCode();
+                Console.WriteLine(response.Content.ReadAsStringAsync());
+                Console.WriteLine($"HTTP GET api/{version}/Tenants/{tenantId}/Users successful");
+            }
+            catch (AggregateException ex)
```

Comment:
Do we want to fail out of the app if we get an Error?

Reply:
I think the test is lazy and assumes there is an exception on failure, so we either need to update the logic there to be better or update this to re-throw exception after logging.

Reply:
I will update the test.